src/algorithms/calorimetry/CalorimeterClusterMerger.cc

```diff
@@ -86,8 +87,8 @@ void CalorimeterClusterMerger::AlgorithmProcess() {
         int nhits = 0;
         auto position = new_clus->getPosition();
         for (auto clus : clusters) {
-          if (false) {
-            LOG_INFO(default_cout_logger) << "   --> Adding cluster with energy: " << clus->getEnergy() << LOG_END;
+          if (m_logger->level() <=spdlog::level::debug) {
```

Comment:
`If` could be removed

Reply:
leaving as close to the original as possible